packages/firestore/src/util/sorted_map.ts

```diff
@@ -526,11 +526,21 @@ export class LLRBNode<K, V> {
 
 // Represents an empty node (a leaf node in the Red-Black Tree).
 export class LLRBEmptyNode<K, V> {
-  key: K;
-  value: V;
-  color: boolean;
-  left: LLRBNode<K, V>;
-  right: LLRBNode<K, V>;
+  get key(): K {
```

Comment:
You could change the return type of all of these to "never"

Reply:
Looks like you're right. Done!